lang/cgilua/Makefile

```diff
@@ -0,0 +1,61 @@
+#
+# Copyright (C) 2016 OpenWrt.org
```

Comment:
This is not correct. You aren't OpenWrt.  Either _drop_ teh copyright outright (my choice) or provide your own name here. (will apply in all places)